### PR TITLE
[Bulk] Validation: Fixed metadata in readme files and other issues

### PR DESCRIPTION
**Pull request by reemsabaw1**
This PR fixes 56 issues in azure-docs-sdk-java.

### CODE REVIEW

**Comment by reemsabaw1** [comment]
@sima-zhu @weidongxu-microsoft @joshfree @backwind1233

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Comment by weidongxu-microsoft** [comment]
I am good with the changes. Just I do not know whether the change on metadata will be overwritten in next release. @sima-zhu for confirmation.

File: docs-ref-services/latest/resourcemanager-iothub-readme.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
 ---
 title: Azure Resource Manager IotHub client library for Java
+description: This package contains Microsoft Azure SDK for IotHub Management SDK. Use this API to manage the IoT hubs in your Azure subscription.
 keywords: Azure, java, SDK, API, azure-resourcemanager-iothub, iothub
-author: ramya-rao-a
-ms.author: ramyar
+author: sima-zhu
```
Comment (joshfree) [inline docs-ref-services/latest/resourcemanager-iothub-readme.md:5]: this isn't correct.  please match the owners from the .github/CODEOWNERS file in the azure-sdk-for-java repo

Comment (joshfree) [reply docs-ref-services/latest/resourcemanager-iothub-readme.md:5]: /cc @weidongxu-microsoft

File: docs-ref-services/latest/resourcemanager-iothub-readme.md [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
+ms.author: sizhu
```
Comment (joshfree) [inline docs-ref-services/latest/resourcemanager-iothub-readme.md:6]: is this a required field?  I wish we didn't leak people's email addresses in these documents

File: docs-ref-services/latest/resourcemanager-mediaservices-readme.md [hunk 3]
```diff
@@ -1,14 +1,13 @@
 ---
 title: Azure Resource Manager MediaServices client library for Java
 keywords: Azure, java, SDK, API, azure-resourcemanager-mediaservices, mediaservices
+description: This package contains Microsoft Azure SDK for MediaServices Management SDK. This Swagger was generated by the API Framework.
 author: joshfree
```
Comment (joshfree) [inline docs-ref-services/latest/resourcemanager-mediaservices-readme.md:5]: all of the author and ms.author fields should be seeded from the .github/CODEOWNERS file in the azure-sdk-for-java repo.

File: docs-ref-services/latest/resources.md [hunk 4]
```diff
@@ -2,13 +2,11 @@
 title: Azure Resource Manager libraries for Java
 description: Reference documentation for the Java Resource Manager libraries 
 keywords: Azure, Java, SDK, API, resource groups, arm, resource manager
-author: ramya-rao-a
-ms.author: ramyar
+author: sima-zhu
+ms.author: sizhu
```
Comment (joshfree) [inline docs-ref-services/latest/resources.md:6]: line-7 what does the manager field here mean?

**Review by joshfree — CHANGES_REQUESTED** [review]
I don't think any of the author fields are correct

**Pull request closed without merging**